src/FsCheck/Runner.fs

```diff
@@ -59,10 +63,11 @@ type Config =
       ///overridden)
       Arbitrary     : list<Type>
       ///A custom test runner, e.g. to integrate with a test framework like xUnit or NUnit. 
-      Runner        : IRunner }
+      Runner        : IRunner
+      ///If set, inputs for property generation and property evaluation will be runned in parallel. 
+      ParallelRunConfig : ParallelRunConfig option}
```

Comment:
First I've added `preserveOrdering :bool` and `algorithm :int`. But benchmarks eliminated all algorithms except current one and preserving ordering was suitable only for plinq. If you have no other options in mind I can leave just bool flag here.